tests/Microsoft.Identity.Test.LabInfrastructure/LabServiceApi.cs

```diff
@@ -161,5 +161,17 @@ public async Task<string> GetUserSecretAsync(string lab)
             string result = await SendLabRequestAsync(LabApiConstants.LabUserCredentialEndpoint, queryDict).ConfigureAwait(false);
             return JsonConvert.DeserializeObject<LabCredentialResponse>(result).Secret;
         }
+
+        public async Task<string> GetMSIHelperServiceTokenAsync()
```

Comment:
Isnt this the same token we are already using for other things? maybe it is better to make the token a property or something that can be accessed when needed rather than making a specific method for MSI?

Reply:
it's the same app but since the service is not hosted under the same lab api, added a different secret so the resources do not share the same secret. This way external teams can use the service with a different secret